pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -365,7 +365,7 @@
     private boolean loadBalancerSheddingEnabled = true;
     // Load shedding interval. Broker periodically checks whether some traffic should be offload from some over-loaded
     // broker to other under-loaded brokers
-    private int loadBalancerSheddingIntervalMinutes = 5;
+    private int loadBalancerSheddingIntervalMinutes = 1;
```

Comment:
nit: this changes the default value. is that okay for any existing users?

Reply:
I think it is safe to change this value because that is only the frequency for checking for overloaded brokers. 5 minutes can be a very long time for that.